src/Speckle.Objects/Geometry/Arc.cs

```diff
@@ -285,21 +145,19 @@ public List<double> ToList()
   /// <returns>A new <see cref="Arc"/> with the values assigned from the list.</returns>
   public static Arc FromList(List<double> list)
```

Comment:
What did we say we'd do here r.e. v2

Reply:
decided to keep the list length the same as in V2 and just pass in dummy values for properties that were marked obsolete